src/main/scala/beam/utils/scenario/urbansim/censusblock/entities/InputPlanElement.scala

```diff
@@ -16,8 +16,8 @@ case class InputPlanElement(
 
 object InputPlanElement extends EntityTransformer[InputPlanElement] {
   override def transform(m: util.Map[String, String]): InputPlanElement = {
-    val personId = getIfNotNull(m, "person_id")
-    val planElementIndex = getIfNotNull(m, "PlanElementIndex").toDouble.toInt
+    val personId = getIfNotNull(m, "person_id").split("\\.").apply(0)
```

Comment:
What about the case where it is less than 0 without a leading 0?

Reply:
Here is how input `plan.csv` looks like:
![image](https://user-images.githubusercontent.com/5107562/88099136-f99bfe80-cbc4-11ea-9fc1-28d1364dbf92.png)
So person_id is represented as float, but agree, splitting is not the best way to deal